docs/misc/config.md

```diff
@@ -63,64 +63,68 @@ The values themselves can be obtained using `ConfigValue#get`. The values are ad
-  * Class Type: `List<T>`
-  * Method Name: `#defineList`, `#defineListAllowEmpty` if list can be empty
-  * Additional Components:
-    * A validator to make sure a deserialized element from the list is valid
+    * Description: Value is a list of entries
+    * Class Type: `List<T>`
+    * Method Name: `#defineList`, `#defineListAllowEmpty` if list can be empty
+    * Additional Components:
+      * A validator to make sure a deserialized element from the list is valid
 
 * **Enum Values**
-  * Description: An enum value in the supplied collection
-  * Class Type: `Enum<T>`
-  * Method Name: `#defineEnum`
-  * Additional Components:
-    * A getter to convert a string or integer into an enum
-    * A collection of the allowed values the configuration can be
+    * Description: An enum value in the supplied collection
+    * Class Type: `Enum<T>`
+    * Method Name: `#defineEnum`
+    * Additional Components:
+      * A getter to convert a string or integer into an enum
+      * A collection of the allowed values the configuration can be
 
 * **Boolean Values**
-  * Description: A `boolean` value
-  * Class Type: `Boolean`
-  * Method Name: `#define`
+    * Description: A `boolean` value
+    * Class Type: `Boolean`
+    * Method Name: `#define`
 
 Registering a Configuration
 ---------------------------
 
 Once a `ForgeConfigSpec` has been built, it must be registered to allow Forge to load, track, and sync the configuration settings as required. Configurations should be registered in the mod constructor via `ModLoadingContext#registerConfig`. A configuration can be registered with a given type representing the side the config belongs to, the `ForgeConfigSpec`, and optionally a specific file name for the configuration.
 
-Type   | Loaded | Synced to Client | Stored                         | Default Suffix
-:---:  | :---:  | :---:            | :---:                          | :---
-CLIENT | Client | ❌               | `.minecraft/config`            | `-client`
-COMMON | Both   | ❌               | `.minecraft/config`            | `-common`
-SERVER | Server | ✔️               | `<server_folder>/serverconfig` | `-server`
-
-!!! tip
-    Forge documents the [config types][type] within their codebase.
-
 ```java
 // In the mod constructor with a ForgeConfigSpec CONFIG
 ModLoadingContext.get().registerConfig(Type.COMMON, CONFIG);
 ```
 
-Listening for Configuration Loading/Reloading
----------------------------------------------
+Here is a list of the available configuration types:
+
+Type   | Loaded           | Synced to Client | Stored                         | Default File Suffix
+:---:  | :---:            | :---:            | :---:                          | :---
+CLIENT | Client Side Only | No               | `.minecraft/config`            | `-client`
+COMMON | On Both Sides    | No               | `<run_location>/config`            | `-common`
+SERVER | Server Side Only | Yes               | `<server_folder>/world/serverconfig` | `-server`
```

Comment:
Table column separators needs to be aligned. Additionally, the `SERVER` type still needs to specify the correct path on the client side -- `.minecraft/saves/<level name>/serverconfig`.